bazarr/subtitles/upload.py

```diff
@@ -151,7 +151,7 @@ def manual_upload_subtitle(path, language, forced, hi, media_type, subtitle, aud
     if use_postprocessing:
         command = pp_replace(postprocessing_cmd, path, subtitle_path, uploaded_language, uploaded_language_code2,
                              uploaded_language_code3, audio_language['name'], audio_language['code2'],
-                             audio_language['code3'], 100, "1", "manual", series_id, episode_id)
+                             audio_language['code3'], 100, "1", "manual", "user", "unknown", series_id, episode_id)
```

Comment:
Why not `None, None` here?

Reply:
or empty string...

Reply:
Yes, you could really put anything equivalent to null or the empty string, but
I was using this to format English like sentences, so I thought it flowed better when my message said “uploaded by user” instead of “uploaded by none” or “uploaded by “.
The same rationale for release_info; how it reads in a sentence.